README.md

```diff
@@ -24,91 +24,139 @@ The `laundry` HTTP server provides an REST API and online tool to try out the co
 
 Conversions are performed in single-use disposable Docker containers. The containers are secured, and their runtime is [gVisor](https://gvisor.dev/) `runsc`. It provides an additional layer of isolation for the containers.
 
-## Building
+## Installation instructions
 
-To rebuild the uberjar and docker images, run:
+Three kinds of installation methods are presented in the following subsections. 
 
-    ./rebuild.sh
+- We recommend to start by doing a [Temporary installation for demo purposes](#temporary-installation-for-demo-purposes) so that you can integrate the `laundry` to your systems and processes. 
+- The [Production installation with Docker and gVisor runsc](#production-installation-with-docker-and-gvisor-runsc) outlines the procedures for a common installation on a dedicated server or vm.
+- You can optionally continue to harden the setup with instructions given in [Customized production  installations](#customized-production-installations)
 
-To rebuild just the uberjar:
+### Temporary installation for demo purposes
 
-    lein uberjar
+**System requirements:** Linux or Mac & Docker
 
-## Development environment
+This installation method gives you the option to try out the laundry, integrate it into your systems or just to play around with it. This temporary installation method is **not suitable for production use** as it lacks sandboxing.
 
-    java -jar target/uberjar/laundry.jar
+```sh
+git clone https://github.com/solita/laundry.git
+./laundry/docker-demo/build-and-run.sh
```

Comment:
```suggestion
./laundry/docker-dev/build-and-run.sh
```

Reply:
The `docker-demo/` is correct. Found another spot where I had left the old `docker-dev/` in place. I'll force-push a fixed version of the readme.